src/config/key_bindings.rs

```diff
@@ -90,11 +90,11 @@ pub struct KeyPattern {
     pub modifiers: KeyModifiers,
     /// The actual key, that was pressed
     #[serde(deserialize_with = "deserialize_Keysym")]
-    pub key: u32,
+    pub key: Option<u32>,
```

Comment:
I think we could just annotate `key` with `#[serde(default  = "Option::None")]` and keep the old deserialization code (see below).